<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en-GB">
<head>
	<title>Homepage of Xiang Li</title>
	<meta http-equiv="Content-Type" content="application/xhtml+xml; charset=utf-8" />
	<meta name="description" content="Homepage of Xiang Li" />
	<meta name="keywords" content="Xiang Li" />
	<meta name="robots" content="index, follow" />
	<link rel="stylesheet" type="text/css" href="./screen.css" media="screen" />
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-140374480-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-140374480-1');
</script>
</head>
<body>

<div id="header">
	<h3>Xiang "Shaun" Li, Instructor in Investigation, Massachusetts General Hospital and Harvard Medical School</h3>
	<ul>
		<li> a </li>
		<li><a href="index.html">About Me</a></li>
		<li><a href="publication.html">Publications</a></li>
		<li><a href="projects.html">Projects</a></li>
		
	</ul>
</div>
<div class="colmask leftmenu">
	<div class="colleft">
		<div class="col1">
		<h3 id="datetime">June 23rd, 2020</h3>
		<h4>Our work on deformable U-Net, validated on the RBC segmentation task, "Automated Semantic Segmentation of Red Blood Cells for Sickle Cell Disease", has been accepted by <i>IEEE Journal of Biomedical and Health Informatics</i>.</h4>
		<h4>The link can be found <a href="https://ieeexplore.ieee.org/abstract/document/9122550/" id="external">here</a></h4>
		<h4>This work is an extension of our previous conference presentation at MICCAI 2018, "RBC Semantic Segmentation For Sickle Cell Disease Based on Deformable U-Net".</h4>
		<hr>
		
		<h3 id="datetime">June 22nd, 2018</h3>
		<h4>Two papers accepted by <a href="www.miccai2020.org" id="external">MICCAI 2020:</a></h4>
		<h4>"Discovering Functional Brain Networks with 3D Residual Autoencoder (ResAE)"</h4>
		<h4>and</h4>
		<h4>"Spatiotemporal Attention Autoencoder (STAAE) for ADHD Classification". Both of the works are in collaboration with Dr. Qinglin Dong, a PhD student of Prof. Tianming Liu at the University of Georgia.</h4>
		<hr>
		
		<h3 id="datetime">April 7th, 2020</h3>
		<h4>Our paper, "ASCNet: Adaptive-Scale Convolutional Neural Networks for Multi-Scale Feature Learning", has been awarded  by ISBI 2020 as the 2nd place winner of Best Paper Award</h4>
		<h4>Congratulates to Mo Zhang, the PhD student first author of this paper!</h4>
		<h4><img src="./img/ISBI2020_bestPaper.png" width="600"/></h4>
		<h4>The announcement video can be found <a href="https://www.youtube.com/watch?v=M5GyJJOKkpE" id="arxiv">here</a>.</h4>
		<hr>
		
		<h3 id="datetime">January 17th, 2020</h3>
		<h4>Our work on fMRI single denoising using dictionary learning method, "Dictionary Learning and Sparse Coding-based Denoising for High-Resolution Brain Activation and Functional Connectivity Modeling: A Task fMRI Study", has been accepted by <i>IEEE Access</i>.</h4>
		<h4>The link can be found <a href="https://ieeexplore.ieee.org/abstract/document/8979389/" id="external">here</a></h4>
		<hr>
		
		<h3 id="datetime">January 7th, 2020</h3>
		<h4>Our work on adaptive scale deep learning method has been accepted by <a href="http://biomedicalimaging.org/2020/" id="external">ISBI 2020:</a></h4>
		<h4>"ASCNet: Adaptive-Scale Convolutional Neural Networks for Multi-Scale Feature Learning".</h4>
		<h4>which can also be found on <a href="https://arxiv.org/abs/1907.03241" id="arxiv"> arXiv </a></h4>
		<hr>
		
		<h3 id="datetime">December 24th, 2019</h3>
		<h4>Our review paper on functional neuroimaging, "Functional Neuroimaging in the New Era of Big Data", is accepted by <i>Genomics, Proteomics and Bioinformatics</i>. The link can be found <a href="https://www.sciencedirect.com/science/article/pii/S1672022919301603" id="external">here</a></h4>
		<h4>This work also summarizes my major perspective on the future trend of functional neuroimaging, proposed in my PhD thesis.</h4>
		<h4><img src="./img/GPB_fig.png" /></h4>
		<hr>
		
		<h3 id="datetime">November 7th, 2019</h3>
		<h4>Our paper on RBC multi-label localization and classification from microscopic images, entitled "Multi-label Detection and Classification of Red Blood Cells in Microscopic Images", has been accepted by the <a href="https://sites.google.com/cs.washington.edu/mlcb/presentations/posters" id="external">14th Annual Meeting on Machine Learning in Computational Biology (MLCB'19) </a>, which is co-located with <i>NeurIPS 2019</i>. Its arXiv link can be found <a href="https://arxiv.org/abs/1910.02672" id="arxiv">here</a></h4>
		<hr>
	
		<h3 id="datetime">November 5th, 2019</h3>
		<h4>Our paper on PET image analysis using graph convolution network (GCN), entitled "Predicting Alzheimerâ€™s Disease by Hierarchical Graph Convolution from Positron Emission Tomography Imaging", has been accepted by the <a href="https://www.cse.msu.edu/~derrtyle/dglma/" id="external">IEEE BigData 2019 Workshop of Deep Graph Learning (DGLMA'19)</a>. Its arXiv link can be found <a href="https://arxiv.org/abs/1910.00185" id="arxiv">here</a></h4>
		<hr>
			
		<h3 id="datetime">October 7th, 2019</h3>
		<h4>Our pneumothorax screening paper is covered by <i>The Imaging Wire</i>, link can be found <a href="https://www.theimagingwire.com/2019/10/07/ed-h-index-qure-ai-qa-radiology-corporatization/" id="external" >here</a></h4>
		<hr>
		
		<h3 id="datetime">September 24th, 2019</h3>
		<h4>Our paper on pneumothorax screening based on CT images, "Deep Learning-Enabled System for Rapid Pneumothorax Screening on Chest CT", has been accepted by <i>European Journal of Radiology</i>. Link can be found <a href="https://www.ejradiology.com/article/S0720-048X(19)30342-0/fulltext" id="external" >here</a></h4>
		<h4>In this work we aim at speeding up and re-prioritizing the radiology workflow using deep learning, taking advantage of its speed and high diagnostic positive detection in analyzing medical images such as CT in pneumothorax. We further conducted large-scale inter-observational study for evaluating radiologist performance of pneumothorax diagnosis, and investigated how AI and human performed differently.</h4>		
		<hr>
			
		<h3 id="datetime">August 20th, 2019</h3>
		<h4>Two of our abstracts are accepted by American Heart Association 2019 annual meeting:</h4>
		<h4>"Personalized Treatment for Heart Failure With Preserved Ejection Fraction Using Deep Reinforcement Learning" is accepted as <a href="https://www.abstractsonline.com/pp8/#!/7891/presentation/31376" id="external">oral presentation</a>, using reinforcement learning to optimize the treatment plans of HFpEF patients. Abstract as appeared in <i>Circulation</i> journal can be found <a href="https://www.ahajournals.org/doi/abs/10.1161/circ.140.suppl_1.15036" id="external">here</a></h4>
		<h4>and "Recurrent Neural Network Enhance Phenotyping in Heart Failure With Preserved Ejection Fraction Using Electronic Health Record" is accepted as <a href="https://www.abstractsonline.com/pp8/#!/7891/presentation/26354" id="external">poster presentation</a>, using recurrent neural network to discover HFpEF phenotypes from patients' longitudinal EHR data. Abstract as appeared in <i>Circulation</i> journal can be found <a href="https://www.ahajournals.org/doi/abs/10.1161/circ.140.suppl_1.12835" id="external">here</a></h4>
		<hr>
		
		<h3 id="datetime">June 10th, 2019</h3>
		<h4><a href="https://mmmi2019.github.io/"  id="external">https://mmmi2019.github.io/</a></h4>
		<h4><img src="./img/MMMI2019.png" height=200 /><br />
		This year Quanzheng Li and I from MGH/HMS, Richard Leahy from USC and Bin Dong from PKU are organising the first MICCAI workshop on Multiscale Multimodal Medical Imaging (MMMI 2019). If you work on related areas, we are looking forward to your paper submission. <strike>The submission deadline is July 31th 2019.</strike><br />
		Proceeding of MMMI 2019 as part of the MICCAI 2019 conference proceeding, published as Lecture Notes in Computer Science (LNCS) book series, is now available online at <a href="https://www.springer.com/us/book/9783030379681" id="external">Springer</a>
		</h4>
		<hr>		
		
		<h3 id="datetime">May 27th, 2019</h3>
		<h4>Our paper pulmonary lymph node metastasis modeling using network inference, entitled "Transition Patterns between N1 and N2 Stations Discovered from Data-driven Lymphatic Metastasis Study in Non-Small Cell Lung Cancer", is accepted by 2019 World Conference on Lung Cancer for <a href="https://library.iaslc.org/conference-program?product_id=15&author=&category=&date=&session_type=&session=&presentation=&keyword=Transition%20Patterns%20between&cme=undefined&" id="external">oral presentation</a>.</h4>
		<h4>This work is based on our own network inference modeling method "PathInf", whicn is online at <a href="https://arxiv.org/abs/1810.00839" id="arxiv">arXiv</a></h4>
		<hr>		
		
		<h3 id="datetime">May 11th, 2019</h3>
		<h4>Our paper on spatio-temporal modeling of fMRI data, "4D Modeling of fMRI Data via Spatio-Temporal Convolutional Neural Networks (ST-CNN)", is accepted by <i>IEEE Transactions on Cognitive and Developmental Systems</i>. The link can be found <a href="https://ieeexplore.ieee.org/document/8713897" id="external">here</a></h4>
		<h4>This work is an extension of our previous conference presentation at MICCAI 2018, "Modeling 4D fMRI Data via Spatio-Temporal Convolutional Neural Networks (ST-CNN)".</h4>
		<hr>		
				
		<h3 id="datetime">January 2nd, 2019</h3>
		<h4>Our paper on multi-modal image fusing in a deep learning context, "Deep Learning-based Image Segmentation on Multi-modal Medical Imaging", is accepted by <i>IEEE Transactions on Radiation and Plasma Medical Sciences</i>. The link can be found <a href="https://ieeexplore.ieee.org/document/8599078?arnumber=8599078" id="external">here</a></h4>
		<h4>This work is an extension of our previous conference presentation at ISBI 2018, "Medical Image Segmentation Based on Multi-Modal Convolutional Neural Network: Study on Image Fusion Schemes".</h4>
		<hr>		
				
		<h3 id="datetime">December 18th, 2018</h3>
		<h4>Four papers accepted by <a href="http:/biomedicalimaging.org/2019/" id="external">ISBI 2019:</a><br />
		<h4>"Multi-Size Computer-Aided Diagnosis of Positron Emission Tomography Images Using Graph Convolutional Networks"</h4>
		<h4>This is our first attempt in using graph representation for medical image analysis, trying to tackle the challenge of varying image sizes in deep learning inputs. We found that graph convolution neural networks, which is based on the concept of non-Euclidean convolution operations, achieves superior performance comparing with traditional 3D CNNs.</h4>
		<h4>"3D Regional Shape Analysis of Left Ventricle Using MR Images: Abnormal Myocardium Detection and Classification"</h4>
		<h4>This work summarizes our preliminary results in performing 3D shape analysis of Cardiac MR images, showing that shape spectral analysis based using multilinear principal component analysis can obtain group-wise shape-based image marker for classifying heart failure vs. normal controls, and different heart failure subtypes.</h4>
		<h4>"Novel Radiomic Features Based on Graph Theory for PET Image Analysis"</h4>
		<h4>In this work we developed new graph-based radiomics features, which capture better intratumoral heterogeneity from PET images comparing with traditional, image-derived texture features.</h4>		
		<h4>"Automated Segmentation of Cervical Nuclei in Pap Smear Images using Deformable Multi-path Ensemble Model"</h4>
		<h4>In this collaborative work with PKU, we developed the Deformable Multipath Ensemble Model (D-MEM) based on an ensembling scheme. It achieved state-of-the-art accuracy for nuclei segmentation in pap smear images for cervical cancer screening. Presentation slides of it could be found <a href="./pdf/ISBI2019_slides.pdf" id="external">here</a></h4><hr>
		
		<h3 id="datetime">October 1st, 2018</h3>
		<h4>Our work on network inference is now online at <a href="https://arxiv.org/abs/1810.00839" id="arxiv">arXiv</a></h4>
		<h4>The PathInf program developed in this work can infer most possible transition paths among variables, only with independently observed instances of a transition process with large missing values.
		The program is tested on simulation data as well as lung caner metastasis data on lung lymph nodes. Performance of PathInf is compared with commonly applied GES (Greedy Equivalence Search) method,
		showing better recovery accuracy, especially for its capability of reducing false positive edges caused by the missing values.</h4>
		<hr>
		
		<h3 id="datetime">August 1st, 2018</h3>
		<h4>Our left ventricle quantification paper is accepted by <a href="http://stacom2018.cardiacatlas.org/" id="external">STACOM workshop</a> of MICCAI 2018:</h4>
		<h4>"Multi-Estimator Full Left Ventricle Quantification through Ensemble Learning", link can be found <a href="https://link.springer.com/chapter/10.1007/978-3-030-12029-0_49" id="external">here</a> arXiv link can be found <a href="https://arxiv.org/abs/1808.02056" id="arxiv">here</a></h4>
		<hr>
		
		<h3 id="datetime">June 15th, 2018</h3>
		<h4>Our <a href="http://www.sciencedirect.com/science/article/pii/S1053811917309035" id="external">Neuroimage paper</a> "Spatio-temporal modeling of connectome-scale brain network interactions via time-evolving graphs" is covered by Editorial of Dialogues in Clinical Neuroscience</h4>
		<h4>The article, "New ways of understanding brain neurocircuitry", can be found <a href="https://www.dialogues-cns.org/wp-content/uploads/2018/08/EDITORIAL_Thibaut_dcns77.pdf" id="external">here</a></h4>
		<hr>		
		
		<h3 id="datetime">May 25th, 2018</h3>
		<h4>Two papers accepted by <a href="www.miccai2018.org" id="external">MICCAI 2018:</a></h4>
		<h4>"RBC Semantic Segmentation For Sickle Cell Disease Based on Deformable U-Net", arXiv link can be found <a href="https://arxiv.org/abs/1710.08149" id="arxiv">here</a></h4>
		<h4>and</h4>
		<h4>"Modeling 4D fMRI Data via Spatio-Temporal Convolutional Neural Networks (ST-CNN)", which is our collaboration work with Dr. Yu Zhao, a PhD student of Prof. Tianming Liu at the University of Georgia. arXiv link can be found <a href="https://arxiv.org/abs/1805.12564" id="arxiv">here</a></h4>
		<hr>
						
		<h3 id="datetime">May 8th, 2018</h3>
		<h4>Our presentation in ARRS 2018 is covered by auntminnie, the report can be found <a href="https://www.auntminnie.com/index.aspx?sec=sup&sub=aic&pag=dis&ItemID=120691&wf=7612" id="external">here</a></h4>
		<hr>
		
		<h3 id="datetime">March 30th, 2018</h3>
		<h4>Our project on pneumothorax detection is among the four finalist of the 2018 NVIDIA Global Impact Award, detailed information can be found <a href="https://blogs.nvidia.com/blog/2018/02/26/ai-radiology-machine-learning-global-impact-awards/" id="external">here</a></h4>
		<hr>
			
		<h3 id="datetime">February 25th, 2018</h3>
		<h4>Our paper on distributed analytics for fMRI based on rank-1 decomposition and cloud computing, titled "A Distributed Computing Platform for fMRI Big Data Analytics", has been accepted by <i>IEEE Transactions on Big Data</i>, the link can be found <a href="https://ieeexplore.ieee.org/document/8307234" id="external">here</a>
		<hr>
		
		<h3 id="datetime">February 21th, 2018</h3>
		<h4>Two reviews on our JACR paper can be found at <a href="http://www.radiologybusiness.com/topics/artificial-intelligence/become-leaders-ai-radiologists-must-address-variety-challenges" id="external">Radiology Business</a> and <a href="https://www.auntminnie.com/index.aspx?sec=log&URL=https%3a%2f%2fwww.auntminnie.com%2findex.aspx%3fsec%3dsup%26sub%3daic%26pag%3ddis%26ItemID%3d119786" id="external">auntminnie.</a></h4>
		<hr>
		
		<h3 id="datetime">February 5th, 2018</h3>
		<h4>Our perspective paper on the impact of artificial intelligence on radiology is now online at:<i>Journal of the American College of Radiology</i>, the link can be found <a href="http://www.jacr.org/article/S1546-1440(17)31671-X/abstract" id="external">here</a></h4>
		<h4>It has been selected as the Continuing Medical Education (CME) material of the month, for ACR credentials.</h4>
		<hr>
		
		<h3 id="datetime">January 22th, 2018</h3>
		<h4>Tensorflow codes for the 2D and 3D deformable convolution network, associated with our work on <a href="https://arxiv.org/abs/1710.08149" id="arxiv">blood cell semantic segmentation</a>, is now available at my GitHub page.</h4>
		<hr>
		
		<h3 id="datetime">December 22th, 2017</h3>
		<h4>Our work on multi-modal image fusion analysis has been accepted by <a href="http://biomedicalimaging.org/2018/" id="external">ISBI 2018:</a></h4>
		<h4>"Medical Image Segmentation Based on Multi-Modal Convolutional Neural Network: Study on Image Fusion Schemes".</h4>
		<h4>which can also be found on <a href="https://arxiv.org/abs/1711.00049" id="arxiv"> arXiv </a></h4>
		<hr>
		
		<h3 id="datetime">November 20th, 2017</h3>
		<h4>Our work on using deep learning for pneumothorax detection on chest CT images has been accepted by <a href="http://www.arrs.org/AM18/" id="external">American Roentgen Ray Society 2018</a> as oral presentation:</h4>
		<h4>"Deep Learning Algorithm for rapid automatic detection of pneumothorax on chest CT"</h4>
		<hr>		
		
		<h3 id="datetime">November 14th, 2017</h3>
		<h4>Another paper on functional brain dynamics has been accepted by <i><a href="http://www.sciencedirect.com/science/article/pii/S1053811917309035" id="external">Neuroimage</a></i>:</h4>
		<h4>In this work we modeled the fMRI signal using dynamic functional networks based on sliding time window approach, then further analyze the patterns of the networks using time-evolving graphs.</h4>
		<hr>
		
		<h3 id="datetime">November 1st, 2017</h3>
		<h4>Our work on fusing multi-modal medical images can be found at <a href="https://arxiv.org/abs/1711.00049" id="arxiv">arXiv</a></h4>
		<h4>We discussed three different fusion schemes for performing supervised learning on medical image analysis. This is a preliminary study from us on how to utilize images from different modalities together to make more accurate and robust image-based decision.</h4>
		<hr>
		
		<h3 id="datetime">October 23rd, 2017</h3>
		<h4>Our work on using deformable U-Net for sickle cell segmentation and classification can be found at <a href="https://arxiv.org/abs/1710.08149" id="arxiv">arXiv</a></h4>
		<h4>The collaboration work with Mo Zhang at Peking University and Mengjia Xu at Northeastern University develop a fully automatic framework for simultaneous cell segmentation and classification (semantic segmentation) on red blood cells microscopic images.</h4>
		<hr>
		
		<h3 id="datetime">July 18th, 2017</h3>
		<h4>Two papers accepted by <a href="mlmi2017.web.unc.edu" id="external">MLMI 2017:</a></h4>
		<h4>"Self-paced Convolutional Neural Network for Computer Aided Detection in Medical Imaging Analysis", arXiv link <a href="https://arxiv.org/abs/1707.06145" id="arxiv">here</a></h4>
		<h4>and</h4>
		<h4>"Dictionary Learning and Sparse Coding-based Denoising for High-Resolution Task Functional Connectivity MRI Analysis", arXiv link <a href="https://arxiv.org/abs/1707.06962" id="arxiv">here</a></h4>
		<hr>	
		
		<h3 id="datetime">May 29th, 2017</h3>
		<h4>Our work on tensor decomposition on sparse and low rank data can be found at <a href="https://arxiv.org/abs/1705.10015" id="arxiv">arXiv</a></h4>
		<h4>The collaboration work with Songting Shi at Peking University offers an integrated framework to perform PARAFAC tensor decomposition on large-scale data with the property of both sparse and low-rank.</h4>
		<hr>
		
		<h3 id="datetime">February 28th, 2017</h3>
		<h4>Our work on using dictionary learning for fMRI signal de-noising is accepted by <a href="http://www.humanbrainmapping.org/OHBM2017/" id="external">OHBM 2017</a></h4>
		<h4>"FMRI Signal Denoising by Dictionary Learning for High-Resolution Functional Connectivity Inference"</h4>
		<h4>The MATLAB package for de-noising (both dictionary learning-based and non-local-mean-based) will be publicly available soon.</h4>
		<hr>
		
		<h3 id="datetime">January 8th, 2017</h3>
		<h4>Two papers accepted by <a href="biomedicalimaging.org/2017/" id="external">ISBI 2017:</a></h4>
		<h4>"Template-guided Functional Network Identification via Supervised Dictionary Learning"</h4>
		<h4>This work is based on the r1DL model previously proposed in my KDD2016 paper. It has shown fast and accurate performance for identifying target functional networks given a set of pre-defined templates from fMRI signals.</h4>
		<h4>"Exploring Human Brain Activation via Nested Sparse Coding and Functional Operators"</h4>
		<h4>The collaboration work with Shu Zhang, a PhD student of Prof. Tianming Liu, proposed a highly novel framework which characterizes functional networks as the results of "functional operators" inside the brain, which offers a brand new perspective for brain functional decoding.</h4>
		<hr>	
		
		<h3 id="datetime">October 13th, 2016</h3>
		<h4>Invited talk on the International Workshop on Big Data Neuroimaging Analytics for Brain and Mental Health at the <a href="http://wibih.unomaha.edu/bih/" id="external">2016 International Conference on Brain Informatics and Health</a>. </h4>
		<hr>
		
		<h3 id="datetime">September 14th, 2016</h3>
		<h4>Joined the Clinical Data Science Center at Harvard Medical School and Massachusetts General Hospital as postdoctoral research fellow, under the mentorship of <a href="http://www.massgeneral.org/doctors/doctor.aspx?id=16374" id="external">James H. Thrall, MD</a> and <a href="http://gordon.mgh.harvard.edu/gc/people/faculty/quanzheng-li/" id="external">Dr. Quanzheng Li</a>.</h4>
		<hr>
		
		<h3 id="datetime">June 2nd, 2016</h3>
		<h4>Two papers accepted by <a href="http://www.miccai2016.org/" id="external">MICCAI 2016:</a></h4>
		<h4>"Modeling Functional Dynamics of Cortical Gyri and Sulci"</h4>
		<h4>This work is our preliminary attempt in investigating the differences of functional dynamics between gyri and sulci areas, which are supposed to work in different roles within the brain functional architecture.</h4>
		<h4>"Discover Mouse Gene Coexpression Landscape Using Dictionary Learning and Sparse Coding"</h4>
		<h4>The work from Yujie Li using sparse coding method on the mouse brain gene data obtained surprisingly cleared-edge map of the mouse brain regions.</h4>
		<hr>
		
		<h3 id="datetime">May 12th, 2016</h3>
		<h4>Awarded the Outstanding Graduate Dissertation/Thesis! Thanks to Computer Science Department for the recognition and my advisor, Prof. Tianming Liu, for the mentorship!</h4>
		<hr>
		
		<h3 id="datetime">May 11th, 2016</h3>
		<h4>Paper accepted by <a href="www.kdd.org/kdd2016/" id="external">ACM SigKDD 2016:</a></h4>
		<h4>"Scalable Fast Rank-1 Dictionary Learning for fMRI Big Data Analysis"</h4>
		<h4>The collaboration work across our group, <a href="cs.uga.edu/~squinn/" id="external">Dr. Shannon Quinn</a> and <a href="http://www.yelab.net/" id="external">Dr. Jieping Ye</a> features our solutions towards fMRI big data analysis by leveraging the distributed computation power by Python Spark.</h4>
		<hr>
		
		<h3 id="datetime">March, 17th, 2016</h3>
		<h4>Talk on <a href="http://siamseas2016.uga.edu/index.shtml" id="external">SIAM-SEAS</a> session "Parallel and distributed computing for biomedical imaging". The presentation slide could be found <a href="./pdf/Dr1Dl@SIAM.pptx" id="external">here</a></h4>
		<hr>
		
		<h3 id="datetime">December 23, 2015</h3>
		<h4>Three papers accepted by <a href="biomedicalimaging.org/2016/" id="external">ISBI 2016:</a></h4>
		<h4>"Modeling Functional Network Dynamics Via Multi-Scale Dictionary Learning and Network Continuums" (with <a href="http://www.yelab.net/">Dr. Jieping Ye)</a></h4>
		<h4>This work marks the development of "network continnum", a novel concept characterizing the continuous/disruptive dynamics of the functional networks. The model reveals ever-changing spatial patterns of the same networks overtime.</h4>
		<h4>"Multiple-Demand System Identification and Characterization Via Sparse Representations of fMRI Data"</h4>
		<h4>This work investigates the well-known multiple-demand system through a network decomposition approach. The work is in sync with our previous Human Brain Mapping paper "Sparse representation of HCP grayordinate data reveals novel functional architecture of cerebral cortex" which focused on MDS on grayordinates (i.e. cortical surface).</h4>
		<h4>"Identifying Group-Wise Consistent Sub-Networks Via Spatial Sparse Representation of Natural Stimulus fMRI Data" (collaboration work with Cheng Lyu)</h4>
		<h4>This work offers a great tool for the brain network analysis from based on its spatial distribution, including sub-network identification and calculating similarities among different networks.</h4>
		<hr>		
		
		<h3 id="datetime">October 14th, 2015</h3>
		<h4>"Sparse representation of HCP grayordinate data reveals novel functional architecture of cerebral cortex", (collaboration work with Xi Jiang) has been accepted by <i>Human Brain Mapping</i>.</h4>
		<hr>		
		
		<h3 id="datetime">July 20th, 2015</h3>
		<h4>Starting visit at <a href="http://www.yelab.net/" id="external">Prof. Jieping Ye's group</a> at University of Michigan.</h4>
		<hr>
		
		<h3 id="datetime">April 1st, 2015</h3>
		<h4>Our article, "
Holistic Atlases of Functional Networks and Interactions Reveal Reciprocal Organizational Architecture of Cortical Function", has been selected as the feature story of <a href="http://tbme.embs.org/2015/03/29/holistic-atlases-functional-networks-interactions-reveal-reciprocal-organizational-architecture-cortical-function/" id="external"><i>IEEE Transactions on Biomedical Engineering</i></a>.<br />
		<img src="./pdf/TBME-00505-2014-Animation.gif" />
		</h4>
		<hr>
		
		<h3 id="datetime">March 17th, 2015</h3>
		<h4>Guest lecture on UGA CSCI6900: Mining Massive Datasets (at Dr. Shannon Quinn's invitation), the presentation slide could be found <a href="./pdf/Dictionary_Learning_in_Functional_Brain_Imaging.pptx" id="pdf">here</a></h4>
		<hr>
		
		<h3 id="datetime">February 13th, 2015</h3>
		<h4>Thanks to Franklin Foundation for providing the <a href="http://psychology.uga.edu/neuro/awards/index.htm" id="external">travel award</a> to support me for attending ISBI 2015!</h4>
		<hr>
		
		<h3 id="datetime">February 5th, 2015</h3>
		<h4>Two papers accepted by <a href="biomedicalimaging.org/2015/" id="external">ISBI 2015:</a><br />
		"Interactive Exemplar-Based Segmentation Toolkit for Biomedical Image Analysis" (with <a href="http://home.penglab.com/" id="external">Dr. Hanchuan Peng)</a><br />
		and<br />
		"Characterizing and Differentiating Task-Based and Resting State FMRI Signals Via Two-Stage Dictionary Learning"</h4>
		<hr>
		
		<h3 id="datetime">February 3rd, 2015</h3>
		<h4>"Characterizing and Differentiating Task-based and Resting State FMRI Signals via Two-stage Sparse Representations" (collaboration work with Shu Zhang) has been accepted by <i>Brain Imaging and Behavior</i><br />
		</h4>
		<hr>

		<h3 id="datetime">September 29th, 2014</h3>
		<h4>Starting a 3-months visit at Allen Institute of Brain Science, with <a href="http://alleninstitute.org/our-institute/our-team/profiles/zhi-zhou/" id="external">Zhi Zhou</a>, <a href="http://alleninstitute.org/our-institute/our-team/profiles/brian-long/" id="external">Brain Long</a>, Hanbo Chen (also from UGA) and <a href="http://home.penglab.com/" id="external">Hanchuan Peng</a>.</h4>
		<hr>
		
		<h3 id="datetime">September 29th, 2014</h3>
		<h4>"Characterizing and Differentiating Brain State Dynamics via Hidden Markov Models" (collaboration work with Dr. Jinli Ou and Prof. <a href="http://mypage.zju.edu.cn/nonlinearsystems" id="external">Leo Xie</a> at ZJU) has been accepted by <i>Brain Topography</i>.</h4>
		<hr>
		
		<h3 id="datetime">May 13th, 2014</h3>
		<h4>Oral presentation at <a href="http://biomedicalimaging.org/2014/" id="external">ISBI 2014</a>, on the topics of Bayesian network-based change point detection and sparse representation of functional network using DICCCOL.</h4>
		<hr>
		
		<h3 id="datetime">April 15th, 2013</h3>
		<h4>Nominated for the best student paper award of <a href="http://www.biomedicalimaging.org/2013/" id="external">ISBI 2013</a><br /></h4>
		</div>
		<div class="col2">
		<p><img src="./img/xiang.jpg" width="200"></p>
		<h4>Reach me at:</h4>
		<h4>25 New Chardon St, 449A, Boston, MA, 02114<br />
		xiangli.shaun{at}gmail.com</h4>
		<h4>My <a href="./pdf/CV_xiang.pdf" id="pdf" target="_blank">CV</a></h4>
		<h4>My <a href="https://researchers.mgh.harvard.edu/profile?profile_id=15451263" id="external" target="_blank">MGH Researcher</a> page</h4>
		<h4>My <a href="https://scholar.google.com/citations?user=MjkwwiQAAAAJ&hl=en" id="external" target="_blank">Google Scholar</a> page</h4>
		<h4>My <a href="https://publons.com/author/1570893/xiang-li" id="external" target="_blank">Publons<a> page</h4>
		<h4>My <a href="https://www.scopus.com/authid/detail.uri?authorId=27168120000" id="external" target="_blank">Scopus author</a> page</h4>		
		<h4>My <a href="https://arxiv.org/a/li_x_10.html" id="external" target="_blank">arXiv author id</a> page</h4>		
		<h4>My ORCID page</h4>
		<a href="https://orcid.org/0000-0002-9851-6376" target="orcid.widget" rel="noopener noreferrer" style="vertical-align:top;"><img src="https://orcid.org/sites/default/files/images/orcid_16x16.png" style="width:1em;margin-right:.5em;" alt="ORCID iD icon">orcid.org/0000-0002-9851-6376</a>
		<h4><a href="https://www.camca.mgh.harvard.edu/" id="external" target="_blank">Homepage</a> of the Center for Advanced Medical Computing and Analysis (CAMCA) at HMS/MGH, where I'm currently working at.</h4>
		<h4><a href="http://caid.cs.uga.edu/" id="external" target="_blank">Homepage</a> of the Cortical Architecture Imaging and Discovery (CAID) lab, where I previous joined at UGA computer science department.</h4>
		<h4><a href="http://braindecodingproject.org/" id="external" target="_blank">Homepage</a> of my previous collaboration work with Dr. Joe Tsien at GRU, the Brain Decoding Project.</h4>
		<h4><a href="http://http://home.penglab.com/" id="external" target="_blank">Homepage</a> of the 3D visualization platform I've been previously working at, the Vaa3D system by Dr. Hanchuan Peng.</h4>
		<!-- Column 2 end -->
		</div>
	</div>
</div>
<div id="footer">
Last updated: November, 2019
</div>

</body>
</html>
